scripts/run_e2e_tests.py

```diff
@@ -532,6 +538,17 @@ def main(args=None):
         sys.exit(1)
     setup_and_install_dependencies(parsed_args.skip_install)
 
+    # Redis-cli is only required in a development environment.
+    python_utils.PRINT('Starting Redis development server.')
+    # Start the redis local development server. Redis doesn't run on
+    # Windows machine.
```

Comment:
machine --> machines

Reply:
Oh haha, I actually caught this in my final pass over the PR. Already fixed.